CoreInteractiveGraph/src/au/gov/asd/tac/constellation/graph/interaction/animation/FlyingAnimation.java

```diff
@@ -49,66 +49,75 @@
 
     @Override
     public void initialise(GraphWriteMethods wg) {
+        // dont initilise the animation if there is less than 2 vertexs
+        if (wg.getVertexCount() <= 1) {
```

Comment:
More curious if the decision to not animate on on 1 node was fixing a bug or purely a design decision? If It's the latter, while I would anticipate it's unlikely someone would want to fly-through on a graph with only one node, I think I would prefer to leave that decision to the user rather than take it out of their hand

Reply:
It was to fix a bug, constellation would crash if there was only 1 node on the fly though. I'm sure there is probably a more appropriate solution that stopped it crashing on 1 node however I figured this was a quicker and much easier fix.

Reply:
Yeah that's alright. Figured I'd check